tests/CMakeLists.txt

```diff
@@ -11,29 +11,23 @@
 
 cmake_minimum_required (VERSION 3.0)
 
-
```

Comment:
Please restore the file to its original version.
You could do this with git checkout:
https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision